### DISCUSSION TITLE
why ** Legacy widget ** is not showing inside ( pages/posts) editor ?

### DISCUSSION THREAD

**Discussion by msabryamin** [WordPress/gutenberg#35159] [Q&A] [answered]
hi , is there any way to show ( legacy widget) inside pages/posts editor ... 

it is only showing in (sidebar) .. 

can you help ?

**Comment by talldan** [comment, marked as answer]
@msa138 I don't think this is supported right now. My understanding is that this limitation was kept when the block-based widget editor was introduced because it wasn't previously possible to insert widgets into posts.

@noisysocks or @kevin940726 may be able to elaborate on this more.

**Reply by draganescu** [reply]
I believe @talldan 's answer is correct. Widgets have historically been reserved for widget areas and there is unclear what bringing them over to posts and pages would add, considering that those which would make sense in the content are implementing functionality handled by core blocks already.

**Reply by kevin940726** [reply]
Yes, this was being discussed in https://github.com/WordPress/gutenberg/issues/24900 for more info.

**Comment by noisysocks** [comment]
Hi @msa138. We don't enable this by default because it requires loading the widget script and style assets into the block editor which comes with a performance penalty.

It is possible for site maintainers to enable this, though. We have some documentation on how to do this here.

https://developer.wordpress.org/block-editor/how-to-guides/widgets/legacy-widget-block/#using-the-legacy-widget-block-in-other-block-editors-advanced

**Reply by Gabriel-Llanos** [reply]
Good morning, a question. By just adding the code from "Using the Legacy Widget block in other block editors (Advanced)" in the functions.php file, will the Legacy Widget appear and work?
I tried it, and it lists all the widgets, but I can't modify the values like the widget title or other values that should be editable, as in the Widgets section. How can I solve this?

**Comment by Gabriel-Llanos** [comment]
Did you solve it? I'm still facing the same issue.
